test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -80,8 +80,8 @@ public void ParentItems_SetsItemsProperty()
 
             // Assert
             Assert.Same(content1, content2);
-            Assert.Equal(expectedContent, content1);
-            Assert.Equal(expectedContent, content2);
+            Assert.Equal(expectedContent, content1.ToString());
+            Assert.Equal(expectedContent, content2.ToString());
```

Comment:
correcting myself: it's fine to override `ToString()` e.g. to help when debugging.  but that isn't for our code.